push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/GCMPushNotificationSender.java

```diff
@@ -80,6 +81,10 @@ public void sendPushMessage(Variant variant, Collection<String> tokens, UnifiedP
         gcmBuilder.addData("alert", message.getAlert());
         gcmBuilder.addData("sound", message.getSound());
         gcmBuilder.addData("badge", "" + message.getBadge());
+        gcmBuilder.priority(message.getPriority() ==     Priority.HIGH ? 
```

Comment:
> Should that comment be in the sender, on the setter, or in some docs somwehere?

I think right about the `gcmBuilder.priority(.....` call in the `GCMPushNotificationSender` class is perfect.

I think the